Collections/collections_home_task.py

```diff
@@ -0,0 +1,458 @@
+"""
+Home task for Module 2 - Collections
+
+Write a code, which will:
+
+1. create a list of random number of dicts (from 2 to 10)
+
+dict's random numbers of keys should be letter,
+dict's values should be a number (0-100),
+example: [{'a': 5, 'b': 7, 'g': 11}, {'a': 3, 'c': 35, 'g': 42}]
+2. get previously generated list of dicts and create one common dict:
+
+if dicts have same key, we will take max value, and rename key with dict number with max value
+if key is only in one dict - take it as is,
+example: {'a_1': 5, 'b': 7, 'c': 35, 'g_2': 42}
+Each line of code should be commented with description.
+
+Commit script to git repository and provide link as home task result.
+"""
+import random  # Import of random for generation of random numbers and letters in dictionaries
+import string  # Import of string for operations with keys of dictionaries
+
+# Create a list of random number of dicts (from 2 to 10)
+list_of_random_dictionaries = []  # Create empty list
+for i in range(0, random.randint(2, 10)):  # FOR statement with randon number of iterations between 2 and 10
+    sample_dictionary = {random.choice(string.ascii_lowercase): random.randint(0, 100)
+                         for y in range(0, random.randint(0, 100))}  # Generation of random dictionary
```

Comment:
```suggestion
                         for _ in range(0, random.randint(0, 100))}  # Generation of random dictionary
```
If you do not use iterable value you can omit it using underscore like this

Reply:
Fixed.